### PR TITLE
[OpenVINO-EP] Fixes OpenVINO-EP build on windows

### PR DESCRIPTION
**Pull request by MaajidKhan**
**Description:**
[FIX] Openvino EP build on windows.

**Motivation and Context:**
The issue is wchar_t is UTF-16 on windows while on other platforms
such as Linux and MacOS, wchar_t is UTF-32.

so wide Unicode string has to be converted to an UTF8 string
for sure on windows else the compiler will throw an error.

This commit fixes this issue.

### CODE REVIEW

File: onnxruntime/test/perftest/ort_test_session.cc [hunk 1]
```diff
@@ -73,7 +85,12 @@ OnnxRuntimeTestSession::OnnxRuntimeTestSession(Ort::Env& env, std::random_device
     std::string device_id = ""; // [enable_vpu_fast_compile]: Fast-compile may be optionally enabled to speeds up the model's compilation to VPU device specific format.
     size_t num_of_threads = 8; // [num_of_threads]: Overrides the accelerator default value of number of threads with this value at runtime.
 
-    std::istringstream ss(performance_test_config.run_config.ep_runtime_config_string);
+    #ifdef _MSC_VER
+    std::string ov_string = utf8_encode(performance_test_config.run_config.ep_runtime_config_string);
```
Comment (jywu-msft) [inline onnxruntime/test/perftest/ort_test_session.cc:89]: can we just use ToMBString() in common.h ?
see https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/common/str_helper.cc#L9

File: onnxruntime/test/perftest/ort_test_session.cc [hunk 2]
```diff
@@ -12,6 +12,18 @@
 namespace onnxruntime {
 namespace perftest {
 
+#ifdef _MSC_VER
+// Convert a wide Unicode string to an UTF8 string
+std::string utf8_encode(const std::wstring &wstr)
```
Comment (jywu-msft) [inline onnxruntime/test/perftest/ort_test_session.cc:17]: remove and use https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/core/common/str_helper.cc#L9  ?

Comment (MaajidKhan) [reply onnxruntime/test/perftest/ort_test_session.cc]: @jywu-msft  Done. Even verified the new changes locally on linux and windows builds as well. It is working fine.

File: onnxruntime/test/perftest/ort_test_session.cc (hunk 1 above)
Comment (MaajidKhan) [reply onnxruntime/test/perftest/ort_test_session.cc]: Done

**Review by HectorSVC — APPROVED** [review]
:shipit:

**Pull request merged**